src/main/webapp/app/admin/lecture-transcription-ingestion/lecture-transcription.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable, inject } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { map } from 'rxjs/operators';
+import { Observable } from 'rxjs';
+
+@Injectable({ providedIn: 'root' })
+export class LectureTranscriptionService {
+    private httpClient = inject(HttpClient);
+
+    ingestTranscription(courseId: number, lectureId: number, lectureUnitId: number): Observable<boolean> {
+        return this.httpClient
+            .put(
+                `api/lecture/courses/${courseId}/lectures/${lectureId}/lecture-unit/${lectureUnitId}/ingest-transcription`,
+                {},
+                {
+                    observe: 'response',
+                },
+            )
+            .pipe(map((response) => response.status == 200));
```

Comment:
isn't it that a 4xx/5xx raises a `HttpErrorResponse` which causes the stream to error out and not invoke the map-lambda? I think one needs to explicitly catch it:
```
pipe(
  map((response) => response.status === 200),
  catchError(() => of(false))
)
```